.vscode/cspell.json

```diff
@@ -106,6 +106,13 @@
         "warnaserror"
       ]
     },
+    {
+      "filename": "**/sdk/analysisservices/**/*.cs",
+      "words": [
+        "Dmts",
+        "MSAL"
```

Comment:
I didn't find `MSAL` in `*.cs` files. Do we still need it?

Reply:
fixed